SkeletonViewCore/Sources/Internal/UIKitExtensions/UIView+AssociatedObjects.swift

```diff
@@ -88,7 +88,7 @@ extension UIView {
     }
 
     var _skeletonableCornerRadius: Float {
-        get { return ao_get(pkey: &ViewAssociatedKeys.skeletonCornerRadius) as? Float ?? 0.0 }
+        get { return ao_get(pkey: &ViewAssociatedKeys.skeletonCornerRadius) as? Float ?? Float(SkeletonViewAppearance.shared.skeletonCornerRadius) }
```

Comment:
I think you don't have to cast this property, right?

Reply:
Yes, you are right, I have remove the casting in 364ae02, please take a look